app/jsx/components/MarqueeComp.jsx

```diff
@@ -41,16 +53,20 @@ class MarqueeComp extends React.Component {
               <header>
                 <h2>About</h2>
               </header>
-              {about_block} 
+              <div className="o-columnbox__truncate2">
```

Comment:
I added this class - strictly to get truncation working for dealbreaker mtg. It's not quite there yet.